website/alicloud.erb

```diff
@@ -544,6 +544,15 @@
                     </ul>
                 </li>
 
+                <li<%= sidebar_current("docs-alicloud-resource-emr-cluster") %>>
+                     <a href="#">EMR Cluster Resources</a>
```

Comment:
I think the line 548 should be `<a href="#">E-MapReduce Resources</a>`

Reply:
> I think the line 548 should be `<a href="#">E-MapReduce Resources</a>`

to be updated.